### ISSUE TITLE
[Feature request] Windows key to open programs in task bar.

### ISSUE THREAD

**Issue by androidnisse** [budgie-desktop/budgie-desktop#1076]
Operating System: Solus 2017.04.18.0
Budgie version: v10.3.1-404-g128a50a

The issue encountered

In Windows when you press the super key+1-9 the pinned program on that spot in the taskbar opens or you change to it if it is already open. In budgie nothing happens when you do this. Now when you want to open a program in the taskbar or change to one on a specific place you have to click on it with the mouse. To add this feature in budgie would save a lot of mouse clicking.

Steps to reproduce the issue

Press super key+1.

**Comment by bbkane** [comment]
Yes please! I'd also really like it if, when searching for an application from the "start menu", (which you can do by pressing super, then typing the name), it would open the program at the top of the list if you press enter. This would match the behavior in Windows.

**Comment by blackomegax** [comment]
3rded

**Comment by hfel** [comment]
4thed

**Comment by dandv** [comment]
Please see #1649 for a better description of this issue, and for arguments as to why this should not be left up to plugin/applet/extension developers.

Note that Unity and the GNOME bundled with Ubuntu 18.04 also support Super + number keys.

**Comment by speltriao** [comment]
No news on this? This is my main "issue" with the great Budgie Desktop =(

**Comment by TheRedstoneRadiant** [comment]
will this be added?
